File: documentation/backend/playground/java/src/main/java/RandomPlayer.java
import java.io.IOException;
import java.util.List;
import java.util.Random;

public class RandomPlayer implements Player{
    @Override
    public String getBestMove(String fen) {
        try {
            Chess chess = new Chess();
            List<String> moves = chess.getLegalMoves();
            Random random = new Random();
            return moves.get(random.nextInt(moves.size()));
        } catch (IOException e) {
            e.printStackTrace();
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
        return null;
    }
}

File: application/chess/random_player.py
from random import Random

import chess

from application.chess.player import Player


class RandomPlayer(Player):
    def get_best_move(self, fen):
        random = Random()
        board = chess.Board(fen=fen)
        generator = board.legal_moves
        moves = []
        for move in generator:
            moves.append(move)
        return moves[random.randint(0, len(moves) - 1)]
